pinot-query-runtime/src/test/resources/queries/TableExpressions.json

```diff
@@ -0,0 +1,93 @@
+{
+  "where_clause_tests": {
+    "psql": "7.2.2",
+    "tables": {
+      "tbl": {
+        "schema": [
+          {"name": "strCol", "type": "STRING"},
+          {"name": "intCol", "type": "INT"}
+        ],
+        "inputs": [
+          ["foo", 1],
+          ["bar", 2],
+          ["alice", 42],
+          ["bob", 196883]
+        ]
+      }
+    },
+    "queries": [
+      { "sql": "SELECT * FROM {tbl} WHERE intCol > 5" },
```

Comment:
(nit) is this intended to be here or in the selectTests that were added in another PR ?

Reply:
not this is a normal select with WHERE clause